Box.V2.JWTAuth/BoxJWTAuth.cs

```diff
@@ -53,14 +54,77 @@ public BoxJWTAuth(IBoxConfig boxConfig)
             {
                 key = (AsymmetricCipherKeyPair)new PemReader(reader, pwf).ReadObject();
             }
-            var rsa = DotNetUtilities.ToRSA((RsaPrivateCrtKeyParameters)key.Private);
+            // ------------------------------------------------------------------
+            // This is the original call to BouncyCastle
+            // var rsa = DotNetUtilities.ToRSA((RsaPrivateCrtKeyParameters)key.Private);
+            // ------------------------------------------------------------------
+            var rsa = ToRSA((RsaPrivateCrtKeyParameters)key.Private);   // use the local version to override the call to DotNetUtilities
```

Comment:
Why not replacing the implementation on DotNetUtilities.ToRSA() instead of having a new one here?